dwave_networkx/algorithms/independent_set.py

```diff
@@ -6,7 +6,7 @@
 
 
 @binary_quadratic_model_sampler(2)
-def maximum_weighted_independent_set(G, weight=None, sampler=None, **sampler_args):
+def maximum_weighted_independent_set(G, weight=None, sampler=None, Lagrange=2.0, **sampler_args):
```

Comment:
`lagrange` should be lower case https://www.python.org/dev/peps/pep-0008/#function-and-variable-names